perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -93,7 +93,7 @@
     'MaxLatency(ms)': max}
 
 
-flags.DEFINE_string('ycsb_version', '0.9.0', 'YCSB version to use. Defaults to '
+flags.DEFINE_string('ycsb_version', '0.13.0', 'YCSB version to use. Defaults to '
```

Comment:
Do not change the default. ycsb is used in a few other benchmarks. Without checking for regression on those, we should not update the default value.

Reply:
i changed this back to the original, this was a workaround for 0.9.0 access issues.